packages/kit/src/runtime/app/forms.js

```diff
@@ -28,16 +28,24 @@ export function enhance(form, submit = () => {}) {
 	 * @param {{
 	 *   action: URL;
 	 *   result: import('types').ActionResult;
-	 *   reset?: boolean
+	 *   reset?: boolean;
+	 *   invalidateAll?: boolean;
 	 * }} opts
 	 */
-	const fallback_callback = async ({ action, result, reset }) => {
+	const fallback_callback = async ({
+		action,
+		result,
+		reset,
+		invalidateAll: shouldInvalidateAll
```

Comment:
Can we just do

```suggestion
		invalidateAll: shouldInvalidateAll = true
```

and then use `if (shouldInvalidateAll)` directly below?

Reply:
Yeah, I was really just emulating the `reset` param but I might just flip both of them...